### ISSUE TITLE
Add new animations (hacktoberfest)

### ISSUE THREAD

**Issue by DevKosov** [UsmanAhmadSaeed/Animations#40]
For the Hacktoberfest event, we're giving you the opportunity to complete the challenge by contributing new animations.
Please read the guidelines on the readme.md file on how to add animations to this repository.

**Comment by Wodlfvllf** [comment]
Hello sir, I want to work. Is it opened?

**Comment by UsmanAhmadSaeed** [comment]
@Wodlfvllf feel free to contribute. Make sure to check contribution guidelines, and adhere to them as well as coding best practices.

**Comment by Yash-RajSingh** [comment]
> @Wodlfvllf feel free to contribute. Make sure to check contribution guidelines, and adhere to them as well as coding best practices.

Hey, so everybody can contribute right? We don't need to wait to get assigned this issue?

**Comment by UsmanAhmadSaeed** [comment]
@Yash-RajSingh exactly

**Comment by Yash-RajSingh** [comment]
> @Yash-RajSingh exactly

Hey, I've just raised a PR for this issue can you please review it and let me know if there are any changes required?

**Comment by anupriyajain595** [comment]
Hello sir I want to contribute in this issue is it open??

**Comment by DevKosov** [comment]
@anupriyajain595 Yes this is open until the end of the month, you're welcome to contribute.

**Comment by Yash-RajSingh** [comment]
Hey @UsmanAhmadSaeed , I've raised another PR #62 for this issue. Could you please review it?

**Comment by udit9876gupta** [comment]
Hello sir, i want to contribute. Is it opened?

**Comment by UsmanAhmadSaeed** [comment]
@udit9876gupta yes it is.

**Comment by UsmanAhmadSaeed** [comment]
@anupriyajain595 I don't see any PR. Make sure you PR to the base repo.

**Comment by udit9876gupta** [comment]
ok sir

> @udit9876gupta yes it is.

ok sir

**Comment by yashneet-kalra** [comment]
Hi @UsmanAhmadSaeed , I've raised a PR #74 for this issue. Kindly review it.

**Comment by UsmanAhmadSaeed** [comment]
https://tenor.com/view/we-take-credit-debit-or-paypal-pay-me-help-me-im-poor-kimmy-gibbler-andrea-barber-gif-15094049

**Comment by UsmanAhmadSaeed** [comment]
https://media.tenor.com/Tk9YGYPs3JoAAAAd/we-take-credit-debit-or-paypal-pay-me.gif

**Comment by serv-er** [comment]
hello sir is it still opened can I contribute for 2023 hactoberfest?

**Comment by UsmanAhmadSaeed** [comment]
@serv-er yes, this will always remain open, for the foreseeable future years to come. Feel free to contribute. I'll keep an eye for review or labelling until hacktoberfest ends.

**Comment by RishiByte** [comment]
Hey sir can i work on this if it still open for 2025 hacktober i will make sure it is in base repo

**Comment by UsmanAhmadSaeed** [comment]
@RishiByte yes, feel free to contribute.
